### ISSUE TITLE
[Bug Report] Expand Row Button Not Tab Accessible

### ISSUE THREAD

**Issue by rkrasnan** [vuetifyjs/vuetify#9939]
### Environment
**Vuetify Version:** 2.1.13
**Vue Version:** 2.6.10
**Browsers:** Chrome 78.0.3904.108
**OS:** Windows 7

### Steps to reproduce
Start with the focus outside of a table that has an expand row.
Press tab.
Focus never lands on the expand row button.

### Expected Behavior
Focus lands on the expand row button. 
Expand Row aria-label and aria-expanded values are read to the screen reader.
Pressing enter, expands and collapses the expansion area.

### Actual Behavior
Expand Button cannot be tabbed to.

### Reproduction Link
<a href="https://codepen.io/rkrasnan/pen/oNgxBOz?&editable=true&editors=101" target="_blank">https://codepen.io/rkrasnan/pen/oNgxBOz?&editable=true&editors=101</a>

**Comment by johnleider** [comment]
This is fixed on the latest release. Originally resolved in v2.2.10.

This Pull Request is being closed due to inactivity.

If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).
